Hackney.Shared.HousingSearch/Domain/Tenure/TempAccommodationInfo.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections.Generic;
+using System.Linq;
+using Hackney.Shared.HousingSearch.Gateways.Models.Tenures;
+using Hackney.Shared.Tenure.Domain;
+
+namespace Hackney.Shared.HousingSearch.Domain.Tenure
+{
+    public class TempAccommodationInfo
+    {
+        public string BookingStatus { get; set; }
+
+        public static TempAccommodationInfo Create(QueryableTempAccommodationInfo TempAccommodationInfo)
+        {
+            return new TempAccommodationInfo(TempAccommodationInfo);
+        }
+        public TempAccommodationInfo()
+        {
+
+        }
+        private TempAccommodationInfo(QueryableTempAccommodationInfo TempAccommodationInfo)
+        {
+            BookingStatus = TempAccommodationInfo.BookingStatus;
```

Comment:
I think it might be possible for this to be called with `TempAccommodationInfo` being null in which case this will throw an error. `TempAccommodationInfo?.BookingStatus;` might in order here

Reply:
Addressed in [fe855f2](https://github.com/LBHackney-IT/housing-search-shared/pull/73/commits/fe855f20214a3db28781b4299853f71c0c033983).